packaging/android/build_and_publish.sh

```diff
@@ -51,10 +51,7 @@ cd "$shared_dir"
 
 if [ ! "$cache_npm" = "1" ]; then
   echo "Cleaning up main node_modules from previous runs"
-  rm -rf "$shared_dir/node_modules"
```

Comment:
i think this likely really slows things down and isn't necessary with --check-files